README.md

```diff
@@ -73,6 +73,10 @@ The majority of the questions were plucked from an [oksoclap](http://oksoclap.co
 * Consider HTML5 as an open web platform. What are the building blocks of HTML5?
 * Describe the difference between cookies, sessionStorage and localStorage.
 * Can you explain the difference between `GET` and `POST`?
+* How can you define custom HTML Elements? 
```

Comment:
Would `<custom></custom>` suffice? Custom HTML Elements have been possible for years. If you'd like to delve into a discussion with a potential candidate about web components, specifically, you should probably mention them in the question. I don't associate "Custom HTML Elements" with web components or even the Shadow DOM right away.